atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CellLoader.java

```diff
@@ -120,9 +121,9 @@ void loadWithTs(
                     SafeArg.of("totalPartitions", totalPartitions));
         }
 
-        List<Callable<Void>> tasks = new ArrayList<>();
+        List<Callable<Void>> tasks = new ArrayList<>(hostsAndCells.size());
         for (Map.Entry<CassandraServer, List<Cell>> hostAndCells : hostsAndCells.entrySet()) {
-            if (log.isTraceEnabled()) {
+            if (isTraceEnabled) {
```

Comment:
in general case I wouldn't normally pull `log.isTraceEnabled()` out to a local, but this popped in profile for relatively hot loop this will almost always be false, so pulled it out a local final boolean